src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -167,4 +236,20 @@ export class ProjectsSummaryController implements angular.IController {
     $('.catalog-projects-summary-modal-backrop').remove();
   }
 
+  private getRecentlyViewedItems() {
+    // recentItems is an array of uids
+    let recentItems: any = this.RecentlyViewed.getItems();
+
+    // replace uids with IServiceItems
+    let items: any = _.map(recentItems, (uid: any) => {
+      return this.allItems[uid];
```

Comment:
It gets called on recently viewed events, and the listener is set up `$onInit`. If I've never passed in image streams or service classes (or they haven't loaded yet), won't this be a runtime error?

I know we expect them to be passed in, but might be better to add a null guard just in case something doesn't load since it's easy to do.